TombEngine/Game/effects/effects.cpp

```diff
@@ -1255,7 +1255,7 @@ void TriggerDynamicLight(const Vector3& pos, const Color& color, float falloff)
 		color.x * UCHAR_MAX, color.y * UCHAR_MAX, color.z * UCHAR_MAX);
 }
 
-// Deprecated. Use above version instead.
+// DEPRECATED! DO NOT USE!!
```

Comment:
?

Reply:
If it is deprecated and do not use, why not just delete it?

Reply:
Sezz did that, don't know why this function is used everywhere :x